OOP/OOPS_Session_1/MatrixTest.java

```diff
@@ -0,0 +1,76 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+public class MatrixTest {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void matrixTransposeTest() {
+		int[] input []={{1,2},{3,4}};
+		int[] expected[]={{1,3},{2,4}};
+		boolean b= new MatrixTest().isArrayEqual(expected ,input);
+		assertTrue(b);
+	}
+	public	boolean	isArrayEqual(int []expected[] ,int []actual[])
+	{ 
+		int flag=0;
+		for(int count1=0;count1<2;count1++)
+			for(int count2=0;count2<2;count2++)
+			{
+				if(expected[count1][count2]==actual[count1][count2])
+					flag=0;
```

Comment:
ACTION : No need to use flag variable , you directly return true false from here.